tests/unit/models/citation-style-test.js

```diff
@@ -0,0 +1,12 @@
+import { moduleForModel, test } from 'ember-qunit';
+
+moduleForModel('citation-style', 'Unit | Model | citation style', {
+  // Specify the other units that are required for this test.
+  needs: []
+});
+
+test('it exists', function(assert) {
+  let model = this.subject();
+  // let store = this.store();
```

Comment:
Should this comment be here?

Reply:
It looks like this is from the `ember g model` blueprint, but I will go ahead and remove it